ts/input/tex/Symbol.ts

```diff
@@ -33,7 +33,7 @@ export class Symbol {
   /**
    * @constructor
    * @param {string} _symbol The symbol parsed.
-   * @param {string} char The corresponding translation.
+   * @param {string} _char The corresponding translation.
```

Comment:
Since the underscore is to prevent Typescript complaining about unused arguments, I'm wondering if we want to include the `_` in the variable name or not.  I think of this as not part of the name, but a mark to indicate we know it is not used (similar to a trailing `?` or `=`).  What do you think?  I notice that it isn't added in other places that have underscores.

Reply:
I agree, in particular we have not been consistent in having the underscore in the JSDoc.
I removed all preceding underscores from the JSDoc now.